service/src/main/kotlin/fi/espoo/evaka/vasu/OphQuestion.kt

```diff
@@ -19,32 +19,193 @@ data class OphQuestionOption(
     val name: Map<VasuLanguage, String>
 )
 
-fun getDefaultTemplateContent(lang: VasuLanguage) = VasuContent(
+fun getDefaultTemplateContent(@Suppress("UNUSED_PARAMETER") lang: VasuLanguage) = VasuContent(
```

Comment:
I'm confused, isn't this annotation completely unnecessary?

Reply:
ah yes, now it is, will remove